tests/transports-integrations/run_integration_tests.py

```
@@ -0,0 +1,271 @@
+
+        if verbose:
+            cmd.append("-v")
+        else:
+            cmd.append("-q")
+
+        # Remove integration-specific marker (not needed for file-based selection)
+        # cmd.extend(["-m", integration])
+
+        # Run the tests
+        try:
+            result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)